exts/cesium.omniverse/cesium/omniverse/ui/credits_viewport_controller.py

```diff
@@ -0,0 +1,70 @@
+from .credits_parser import CesiumCreditsParser, ParsedCredit
+from typing import List, Optional, Tuple
+import logging
+import carb.events
+import omni.kit.app as app
+from ..bindings import ICesiumOmniverseInterface
+import omni.ui as ui
+import omni.kit.app
+import json
+from carb.events import IEventStream
+
+
+class CreditsViewportController:
+    def __init__(self, cesium_omniverse_interface: ICesiumOmniverseInterface):
+        self._cesium_omniverse_interface = cesium_omniverse_interface
+        self._logger: Optional[logging.Logger] = None
```

Comment:
No need to declare and set this here, you're setting up the logger below.

Reply:
Ah, I had `__init__` set up where everything is declared as None or [] with type hinting in a block then instantiated later. Not sure where I got that pattern. No need to use it.

Changed.